### PR TITLE
MAUI-1681 Sample added

### PR DESCRIPTION
**Pull request by Lakshminatarajan**
MAUI-1681 Sample added

### CODE REVIEW

File: ListViewMaui/ListViewMaui/ViewModel/ContactsViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+using System.Text;
+
+
+namespace ListViewMaui
+{
+    public class ContactsViewModel : INotifyPropertyChanged
+    {
+        #region Properties
+
+        public ObservableCollection<Contacts> contactsinfo { get; set; }
```
Comment (jayaleshwari) [inline ListViewMaui/ListViewMaui/ViewModel/ContactsViewModel.cs]: change the case property name

**Review by jayaleshwari — CHANGES_REQUESTED** [review]
@Lakshminatarajan , Please address review correction.

File: ListViewMaui/ListViewMaui/MauiProgram.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+	public static MauiApp CreateMauiApp()
+	{
+		var builder = MauiApp.CreateBuilder();
+		builder
+			.UseMauiApp<App>()
+			.ConfigureFonts(fonts =>
+			{
+				fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+			});
+
```
Comment (jayaleshwari) [inline ListViewMaui/ListViewMaui/MauiProgram.cs:16]: @Lakshminatarajan , subscribe syncfusion listview handlers here.

**Review by jayaleshwari — CHANGES_REQUESTED** [review]

File: ListViewMaui/ListViewMaui/MauiProgram.cs (hunk 2 above)
Comment (Lakshminatarajan) [reply ListViewMaui/ListViewMaui/MauiProgram.cs:16]: Added SyncfusionListView handler

File: ListViewMaui/ListViewMaui/ViewModel/ContactsViewModel.cs (hunk 1 above)
Comment (Lakshminatarajan) [reply ListViewMaui/ListViewMaui/ViewModel/ContactsViewModel.cs]: Updated the property name as suggested.

**Review by jayaleshwari — APPROVED** [review]
This is fine

**Pull request merged**